src/commands/crashlytics-mappingfile-generateid.ts

```diff
@@ -0,0 +1,39 @@
+import { Command } from "../command";
+import * as utils from "../utils";
+
+import { fetchBuildtoolsJar, runBuildtoolsCommand } from "../crashlytics/buildToolsJarHelper";
+
+interface CommandOptions {
+  resourceFile: string | null;
+  debug: boolean | null;
+}
+
+interface JarOptions {
+  resourceFilePath: string;
+}
+
+export default new Command("crashlytics:mappingfile:generateid")
+  .description(
+    "Generate a mapping file id and write it to an Android resource file, which will be built into the app."
```

Comment:
```suggestion
    "generate a mapping file id and write it to an Android resource file, which will be built into the app"
```

Following guidance from go/firebase-cli-contributor#create-a-file-for-your-command - please adjust the other options/commands in this PR

Reply:
Is there any way to mention that the mapping file is for deobfuscating stack traces?